.travis.yml

```diff
@@ -2,7 +2,11 @@ language: node_js
 
 matrix:
   fast_finish: true
+  allow_failures:
+    - node_js: 4
   include:
+    - node_js: node
```

Comment:
Will it be `9.0.0` at this moment? If so we might need to replace `7` by `8` since `7` is not supported.

Reply:
Good point.

And its node 9.0.0: https://travis-ci.org/mzgoddard/hard-source-webpack-plugin/jobs/295524042#L464